terms/neuro/study.json

```diff
@@ -254,6 +254,11 @@
             "description": "Strain validation data from the Jax.IU.Pitt MODEL-AD grant",
             "source": "https://www.synapse.org/#!Synapse:syn21595255"
         },
+        {
+            "const": "Jax.IU.Pitt_Verubecestat_5XFAD",
+            "description": "Data from 5XFAD mice treated with verubecestat from the Jax.IU.Pitt MODEL-AD grant",
+            "source": "https://www.synapse.org/#!Synapse:syn21863375"
```

Comment:
source should be the link to the AD portal study card (doesn't exist yet but uses the study folder synID): "https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage?Study=syn21863375"